libs/clients/payment-schedule/src/lib/payment-schedule.ts

```diff
@@ -74,9 +74,7 @@ export class PaymentScheduleAPI extends RESTDataSource {
 
     return {
       ...response.distributionInitialPosition,
-      minPayment: roundToUpperThousand(
-        response.distributionInitialPosition.minPayment,
-      ),
+      minPayment: response.distributionInitialPosition.minPayment,
```

Comment:
No need to spread since the rounding is gone, this could just be

`return response.distributionIntialPostion`

Reply:
Fixed